src/router/index.js

```diff
@@ -21,6 +22,11 @@ const routes = [
     name: 'Profile',
     // add component later
   },
+  {
+    path: '/UserResView',
+    name: 'UserResView',
+    component: UserResView,
+  },
```

Comment:
So close! Instead, just import your UserRes component and change the path to /UserRes. UserResView is unnecessary. 
```suggestion
  {
    path: '/UserRes',
    name: 'UserRes',
    component: UserRes,
  },
```

Reply:
Done